src/app/api.mjs

```diff
@@ -201,3 +203,10 @@ export async function login(data) {
   const response = await client.post(url, data);
   return response.data;
 }
+
+export async function signupDetails(data) {
```

Comment:
Important: this function brakes decoupling. It is coupled to a localStorage. That limits it's reusability. For example, will I be able to call it from node environment to experiment with my ip from shell? That is generally a good question to check decoupling: how will I run this functionality another way. I usually check with shell in mind. 

https://react.dev/learn/keeping-components-pure